suite-native/atoms/src/Sheet/BottomSheet.tsx

```diff
@@ -112,22 +114,28 @@ export const BottomSheet = ({
                                 isCloseDisplayed={isCloseDisplayed}
                                 onCloseSheet={closeSheetAnimated}
                             />
-                            <ScrollView
-                                ref={scrollViewRef.current}
-                                waitFor={
-                                    isCloseScrollEnabled
-                                        ? panGestureRef.current
-                                        : scrollViewRef.current
-                                }
-                                onScroll={scrollEvent}
-                                keyboardShouldPersistTaps="handled"
-                            >
-                                <Animated.View>
-                                    <Box paddingHorizontal="medium" {...boxProps}>
-                                        {children}
-                                    </Box>
-                                </Animated.View>
-                            </ScrollView>
+                            {isScrollable ? (
+                                <ScrollView
+                                    ref={scrollViewRef.current}
+                                    waitFor={
+                                        isCloseScrollEnabled
+                                            ? panGestureRef.current
+                                            : scrollViewRef.current
+                                    }
+                                    onScroll={scrollEvent}
+                                    keyboardShouldPersistTaps="handled"
+                                >
+                                    <Animated.View>
+                                        <Box paddingHorizontal="medium" {...boxProps}>
+                                            {children}
+                                        </Box>
+                                    </Animated.View>
+                                </ScrollView>
+                            ) : (
+                                <Box {...boxProps} style={{ height: '100%' }}>
```

Comment:
this has different style than the box inside scrollview and there is no Animated.View. Is that intended?

Reply:
I think `Animated.View` might be even redudant inside that scrollview, it seems it doesn't have any obvious purpose. Maybe @PeKne may know why it's there?

Reply:
Anyway I created follow up #14503 which could be used to refine this.